core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -499,6 +499,13 @@ package object config {
     .stringConf
     .createOptional
 
+  private[spark] val METRICS_CPU_TIME_DRIVER_ENABLED =
```

Comment:
Does this need a flag -- is there a drawback to enabling it?

Reply:
On second thoughts adding a flag/configuratoni parameter is probably not necessary and can be removed, as the JVMCPU source should be quite safe. We have been using it for the executor instance for a while. Also the implementation takes care of various error conditions that can occur for exmaple with JVMs that don't implement "ProcessCpuTime"